openapi/components/schemas/Quote.yaml

```
@@ -1,357 +1,10 @@
-              type:
-                - 'number'
-                - 'null'
-              format: double
-  paymentTerms:
-    description: Payment terms for the customer which are displayed on the quote.
-    type: string
-  expirationTime:
-    description: |-
-      Date and time when the quote expires. The default expiration time is one month from the time the quote is issued.
-      In a `draft` state, this field may be `null`.
-    type:
-      - 'string'
-      - 'null'
-    format: date-time
-  issuedTime:
-    description: Date and time when the quote is issued.
-    type:
-      - 'string'
-      - 'null'
-    readOnly: true
-    format: date-time
-  acceptedTime:
-    description: Date and time when the quote is accepted.
-    type:
-      - 'string'
-      - 'null'
-    readOnly: true
-    format: date-time
-  rejectedTime:
-    description: Date and time when the quote is rejected.
-    type:
-      - 'string'
-      - 'null'
-    readOnly: true
-    format: date-time
-  canceledTime:
-    description: Date and time when the quote is canceled.
-    type:
-      - 'string'
-      - 'null'
-    readOnly: true
-    format: date-time
-  redirectUrl:
-    description: URL to redirect the customer to when a quote is rejected. The default value is the website URL.
-    type: string
-    format: uri
-  signature:
-    type: object
-    properties:
-      showWrittenSignatureLines:
-        type: boolean
-        default: false
-        description: Specifies whether to show written signature lines.
-      organizationPrintedName:
-        type:
-          - 'string'
-          - 'null'
-        description: Printed name of the organization.
-        default: null
-  shipping:
-    $ref: ./Shipping.yaml
-  tax:
-    $ref: ./Taxes.yaml
-  couponIds:
-    type:
-      - 'array'
-      - 'null'
-    description: |-
-      List of coupons to redeem on the customer and apply to the order when the quote is accepted.
-
-      For more information, see [Coupons](https://www.rebilly.com/docs/settings/coupons-and-discounts/).
-    items:
-      type: string
-      description: Coupon ID.
-  _links:
-    type: array
-    description: Related links.
-    readOnly: true
-    items:
-      type: object
-      properties:
-        href:
-          description: Link URL.
-          type: string
-        rel:
-          description: Type of link.
-          type: string
-          enum:
-            - self
-            - quoteAcceptanceFormUrl
-  _embedded:
-    type: object
-    description: Embedded objects that are requested by the `expand` query parameter.
-    readOnly: true
-    properties:
-      customer:
-        type: object
-      website:
-        type: object
-      order:
-        type: object
+discriminator:
+  propertyName: action
+  mapping:
+    create: ./QuoteCreateOrder.yaml
+    change: ./QuoteChangeOrder.yaml
+    reactivate: ./QuoteReactivateOrder.yaml
+anyOf:
```

Discriminator property is required and mutually exclusive
```suggestion
oneOf:
```